dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -75,19 +111,78 @@ def version():
 
 
 @option_verbose
-@main.command()
-def shell(verbose: bool):
+@main.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    ),
+)
+@option_python_version
+@option_backend
+@click.option('--integration', type=click.Choice(ALLOWED_INTEGRATIONS), multiple=True)
+@click.option('-L', '--build-cache-local', is_flag=True)
+@click.option('-U', '--build-cache-pulled', is_flag=True)
+@click.option('-X', '--build-cache-disabled', is_flag=True)
+@click.option('--production-image', is_flag=True)
```

Comment:
@potiuk wrt [this](https://github.com/apache/airflow/pull/21848) issue, we don't need the `--production-image` flag and any process related to that flag, am I right?

Reply:
Correct. This will be spearate command ("build-prod-image").

Reply:
BTW. You have now some exercise in conflict resolution as I merged the fix for storing python version :). Happy to help if needed.

Reply:
Let me try first. If i didn't get success, i will get your help.